monkestation/code/datums/traits/negative.dm

```diff
@@ -0,0 +1,16 @@
+/datum/quirk/jailbird
+	name = "Jailbird"
+	desc = "You're a wanted criminal! You start the round set to arrest for a random crime."
+	value = -1
+
+/datum/quirk/jailbird/post_add() //Deliberately delayed to allow for the actual data_core entry to be created
+	. = ..()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/quirk_crime	= pick(world.file2list("monkestation/strings/random_crimes.txt"))
+	to_chat(H, "<span class='boldnotice'>You are on the run for your crime of: [quirk_crime]!</span>")
+	var/crime = GLOB.data_core.createCrimeEntry(quirk_crime, "Galactic Crime Broadcast", "[pick(world.file2list("monkestation/strings/random_police.txt"))]", "[(rand(9)+1)] [pick("days", "weeks", "months", "years")] ago", (rand(100)*10))
+	var/perpname = H.get_id_name() //Strangely doesn't work unless done in this exact way.
```

Comment:
get_id_name gets the name on their ID, I'm assuming. Im not sure if we have any traits that gets rid of the ID but if we do/get one that does it will probably cause issues.

Reply:
Very possibly. 
The only issue is that I simply couldn't get it to work by feeding the proc the name directly with H.name OR H.real_name

Reply:
Look into what get_id_name returns thats different from the others

Reply:
Issue found and fixed.